third_party/bazel/acl/acl_makefile.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/Makefile b/Makefile
+new file mode 100644
+index 0000000..bd9a183
+--- /dev/null
++++ b/Makefile
+@@ -0,0 +1,10 @@
++all:
```

Comment:
A patch works!
But is it possible to use `build_data` of `make` rule?

Reply:
I think it will not work, I tried adding it in `build_data`, however when `make` is called, the `Makefile` cannot be found under the acl's root code dir.